go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/bsm/redislock v0.9.4
 	github.com/coreos/go-oidc/v3 v3.9.0
 	github.com/fsnotify/fsnotify v1.7.0
-	github.com/ghodss/yaml v1.0.1-0.20220118164431-d8423dcdf344
+	github.com/ghodss/yaml v1.0.0
```

Comment:
Not sure what's going on here. Maybe we should switch to goccy/go-yaml or just the default go-yaml?